packages/machine/test/unit/cf-operations/set-state.spec.ts

```diff
@@ -79,12 +37,20 @@ describe("CfOpSetState", () => {
     const digest = op.hashToSign();
     const vra1 = TEST_SIGNING_KEYS[0].signDigest(digest);
     const vra2 = TEST_SIGNING_KEYS[1].signDigest(digest);
-    const sig1 = new Signature(vra1.recoveryParam as number, vra1.r, vra1.s);
-    const sig2 = new Signature(vra2.recoveryParam as number, vra2.r, vra2.s);
+    const sig1 = new cf.utils.Signature(
```

Comment:
It should probably be possible to pass in `vra1` directly into the constructor rather than decomposing it. Perhaps using spread operator like `new Signature(...vra1)`